snap/snapcraft.yaml

```diff
@@ -24,40 +25,47 @@ layout:
 apps:
   hotsos:
     command: bin/hotsos
+
 parts:
-  hotsos-cli:
-    plugin: dump
-    source: .
-    override-build: |
-      rm -rf tests
-      git rev-parse --short HEAD > repo-info
-      snapcraftctl build
-    organize:
-      defs: etc/hotsos/defs
-    build-packages: [git]
   hotsos:
-    plugin: python
+    plugin: dump
     source: .
-    stage-packages: [coreutils, bsdmainutils, systemd, dpkg, python3-yaml,
-                     python3-simplejson, python3-click]
-    python-packages:
-      - cryptography==3.4.8
-      - wheel
-      - pip
-      - setuptools
-    requirements:
-      - requirements.txt
+    build-packages:
+      - git
+      - python3-pip
+      - python3-venv
+      - rsync
+    stage-packages:
+      - bsdmainutils
+      - coreutils
+      - dpkg
+      - libc6
+      - libpython3.8-minimal
+      - libpython3.8-stdlib
```

Comment:
if we are going to carry a python version id like to carry the same one the core snap has. In the case of core20 that is 3.8 but then we recently started running all tests using 3.10 which is jammy's version so could we switch core20 -> core22 and bump 3.8 -> 3.10?

Reply:
I am running into issues with switching to core22. It's like linking hell all over :/ What do you think of merging this PR while I keep working on the core22 upgrade. Merging this change here would mean we finally have a working snap.